examples/providerconfig/secret.yaml.tmpl

```diff
@@ -1,12 +0,0 @@
-apiVersion: v1
```

Comment:
Is deletion of this file intentional? Don't you want to follow our convention regarding having a template for the expected secret?

Reply:
Yes. As you said, I will keep it.